icon-two-common-models/src/main/java/ca/bc/gov/open/icon/models/serializers/BooleanConverter.java

```diff
@@ -0,0 +1,21 @@
+package ca.bc.gov.open.icon.models.serializers;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class BooleanConverter {
+    private BooleanConverter() {}
+
+    public static String print(boolean b) {
+        return b ? "1" : "0";
+    }
+
+    public static boolean parse(String s) {
+        try {
+            return s.equals("1");
+        } catch (Exception ex) {
+            log.warn("Bad data received from soap request - invalid boolean: " + s);
+            return false;
```

Comment:
equals will never throw exception, you need to check if the string is 1/0 before equals

Reply:
if s is null or garbage, then equals will throw exception. I do not need to check if the string is 1/0 before the equals.
if s is valid string,   equals will return either true when s is "1" otherwise false.